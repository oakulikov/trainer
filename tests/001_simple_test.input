result,oddF,oddX,oddL
X,2.0,3.5,4.0
F,1.9,3.3,4.1
L,1.85,3.6,4.4