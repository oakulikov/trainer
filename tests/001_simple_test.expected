event_number,result,oddF,oddX,oddL,betF,betX,betL,lossF,lossX,lossL,total,uf,ux,ul,pattern
3,L,1.85,3.60,4.40,16900,6600,7750,31250,23650,0,30000,2,2,0,
2,F,1.90,3.30,4.10,13800,6150,6250,0,14050,29300,20000,0,1,1,
1,X,2.00,3.50,4.00,10000,4000,3350,20000,0,13350,10000,1,0,1,